script.js

```diff
@@ -11,4 +11,137 @@ document.querySelectorAll(".nav-link").forEach(n => n.
+    description2: "Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.",
+    img1: "Snapshoot Portfolio (3).png",
+    img2: "Snapshoot Portfolio (3).png",
+    technologies: ["Html", "Css", "Javascript"],
+    liveDemo: "#",
+    sourceCode: "",
+    title1: "Multi-Post Stories",
+    title2: "Multi-Post Stories",
+},
+{
+    description: "Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.",
+    description2: "Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.",
+    img1: "Snapshoot Portfolio.png",
+    img2: "Snapshoot Portfolio.png",
+    technologies: ["Html", "Css", "Javascript"],
+    liveDemo: "#",
+    sourceCode: "",
+    title1: "Facebook 360",
+    title2: "Facebook 360",
+},
+{
+    description: "A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.",
+    description2: "A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.",
+    img1: "Snapshoot Portfolio (1).png",
+    img2: "Snapshoot Portfolio (1).png",
+    technologies: ["Html", "Css", "Javascript"],
+    liveDemo: "#",
+    sourceCode: "",
+    title1: "Uber Navigation",
+    title2: "Uber Navigation",
+}
+]
+
+const section = document.querySelector(".works")
+const popup = document.querySelector(".popup-window")
+
+project.forEach((projects) => {
+    const article = document.createElement('article')
+    const divImg = document.createElement('div')
+    const img = document.createElement('img')
+    const elements = document.createElement('div')
+    const button = document.createElement("button")
+    img.src = `${projects.img1}`
+    article.classList.add('card1')
+    button.classList.add('btn')
+    elements.classList.add('elements1')
+    button.innerText = 'See Project'
+
+    elements.innerHTML += `
+    <h2 class="title">${projects.title1}</h2>
+                        <ul class="highlights">
+                            <li>CANOPY</li>
+                            <li><img class="counter" src="Counter.png" alt="counter"></li>
+                            <li>Back End Dev</li>
+                            <li><img class="counter" src="Counter.png" alt="counter"></li>
+                            <li>2015</li>
+                        </ul>
+                        <p class="description">${projects.description}</p>
+                        <ul class="languages">
+                            <li class="stack">${projects.technologies[0]}</li>
+                            <li class="stack">${projects.technologies[1]}</li>
+                            <li class="stack">${projects.technologies[2]}</li>
+                        </ul>
+    `
+    elements.append(button)
```

Comment:
- [ ] Good job creating the project cards dynamically 👏🏻 , but while doing so, the display got messed up in this section. Please fix the display of the elements so that they don't overlap.

![image](https://user-images.githubusercontent.com/62290545/195361772-51b9581d-2e21-47c1-84ec-95ea1b295b82.png)